roles/linux_preparation/tasks/recore.yml

```diff
@@ -47,7 +47,7 @@
 - name: write dogtag
   copy:
     dest: '/etc/{{ dogtag }}'
-    content: '{{ version.stdout }} {{ build_date.stdout }}'
+    content: '{{ version.stdout }}{{ build_date.stdout }}{{ platform }}'
```

Comment:
This might have consequences during image generation. The image name is pulled from the dogtag and assumes a space is present.

Reply:
Ah. The issue I had was that the space was actually presenting as a newline in the dogtag file itself. By removing the space it showed up as:
```
3.0.3-RC5
2022-01-23
recore
```
Where the original comes out as
```
3.0.3-RC5
 2022-01-23
```
By having it in separate lines it's easier to use scripts that use regex to correctly use the right line, instead of having to cut out the substring from spaces.